app/scripts/controllers/contributions.js

```diff
@@ -62,12 +94,25 @@ angular.module('bulbsCmsApp')
         for (var i in contributions) {
           if (contributions[i] === null || contributions[i].role === undefined) {
             continue;
+          } else {
+
+            if (contributions[i].hasOwnProperty('rate') === 'object') {
```

Comment:
I don't think this is the correct usage of ```hasOwnProperty```. This function [returns a boolean](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty), not a string.